batch/Dockerfile

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
nit: Can you please add the Apache license note at the top of all new files?

BTW, if you prefer "Google LLC" to be switched to something more generic (like "OpenMRS Analytics Team"), please feel free to update everywhere; given that we are using Apache license and everyone is acknowledging the Google Contributor License Agreement, we don't have to use "Google LLC" everywhere. But whatever we pick, let's be consistent.

Reply:
Noted